cgroups/retrying_write.go

```diff
@@ -0,0 +1,21 @@
+package cgroups
+
+import (
+	"errors"
+	"io/ioutil"
+	"os"
+	"syscall"
+)
+
+func retryingWriteFile(path string, data string, mode os.FileMode) error {
+	// Retry writes on EINTR; see:
+	//    https://github.com/golang/go/issues/38033
+	for {
+		err := ioutil.WriteFile(path, []byte(data), mode)
```

Comment:
The `ioutil` package was deprecated. Prefer `os` instead.

```suggestion
		err := os.WriteFile(path, []byte(data), mode)
```